### ISSUE TITLE
Add linting and fix all issues with the code 

### ISSUE THREAD

**Issue by chandrams** [kruize/autotune#275]
Add linting and fix all issues seen with the code.

**Comment by ShishirH** [comment]
We can probably use solarLint (https://www.sonarlint.org/intellij)

I can pick this up if no one else is working on it

**Comment by chandrams** [comment]
Yes, you can pick this up. We can discuss the priority of this during our scrum call. This should be run regularly as part of PR check.
